.github/ISSUE_TEMPLATE.md

```diff
@@ -18,4 +18,7 @@ STOP!! Please don't raise security vulnerabilities here. Head over to https://pi
 🎁 Enhancement
 Please start by describing the problem that you are trying to solve. There may already
 be a solution, or there may be a way to solve it that you hadn't considered.
+
+- Managed dependency upgrade
```

Comment:
Where is the emoji? 🤣

Reply:
@snicoll That was the hardest part, so I intentionally avoided choosing one by using "-" 😅 

Maybe 🙅?